src/pdf.rs

```diff
@@ -17,21 +17,21 @@
 
 //! Create PDFs for printed paper backup
 
-use pandoc;
```

Comment:
How can this work? Below (line 72) I use `pandoc::new()`, how can that work without importing pandoc in this file?

Reply:
external crates are implicitly imported (its the `extern crate <crate>` invocation that's implicit) in 2018 edition